src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -1202,6 +1209,36 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         End Function
 
+        Public Shared Function ClassifyTupleConversion(source As BoundTupleLiteral, destination As TypeSymbol, binder As Binder, <[In], Out> ByRef useSiteDiagnostics As HashSet(Of DiagnosticInfo)) As ConversionKind
+            Dim arguments = source.Arguments
+
+            ' check if the type is actually compatible type for a tuple of given cardinality
+            If Not destination.IsTupleOrCompatibleWithTupleOfCardinality(arguments.Length) Then
+                Return Nothing 'ConversionKind.NoConversion
```

Comment:
Just following the pattern of the usage in numerous other places. I do not know the reason, but usage of Nothing seems to be very intentional.